vibe_code_apps_22/scripts/build_vibe22_rl_validity_report.py

```
@@ -0,0 +1,256 @@
+    p9_path = ROOT / "docs" / "audits" / "figures" / "phase9_campaign.json"
+    p9_txt = "NOT RUN"
+    if p9_path.is_file():
+        p9 = json.loads(p9_path.read_text(encoding="utf-8"))
+        p9_txt = f"{p9.get('status')}: {p9.get('reason')}"
+
+    from eplus_gym.rleplus_path import find_rleplus_root, rleplus_git_sha
+
+    try:
+        rroot = find_rleplus_root()
+        rsha = rleplus_git_sha(rroot)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)